mozregression/cli.py

```diff
@@ -386,13 +386,10 @@ def _convert_to_bisect_arg(self, value):
         try:
             value = parse_date(value)
         except DateFormatError:
-            try:
-                new_value = parse_date(date_of_release(value))
-                self.logger.info("Using date %s for release %s"
-                                 % (new_value, value))
-                value = new_value
-            except UnavailableRelease:
-                pass
+            new_value = parse_date(date_of_release(value))
```

Comment:
I think it's good to log the message here, as it's a bit more friendly than just throwing the exception.